lib/challenge.js

```diff
@@ -61,6 +91,24 @@ var some_pet;
 var Pet;
 // Answer Starts Here
 
+var Pet = function Pet(name, species, breed, noise) {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = []];
+
+};
```

Comment:
Aside from that loose bracket, this is a perfectly good constructor function. Well done!